doc/source/reference/arrays.indexing.rst

```diff
@@ -36,8 +36,8 @@ objects, the :const:`Ellipsis` object, or the :const:`newaxis` object,
 but not for integer arrays or other embedded sequences.
 
 .. index::
-   triple: ndarray; special methods; getslice
-   triple: ndarray; special methods; setslice
+   triple: ndarray; special methods; get slice
```

Comment:
Should be an ndarray method. `getitem` and `setitem` are probably what should be here.